spec/rubocop/cop/bundler/duplicated_group_spec.rb

```diff
@@ -77,6 +77,16 @@
       end
     end
 
+    context 'and a group has a keyword option' do
+      it 'does not register any offenses' do
+        expect_no_offenses(<<~RUBY, 'Gemfile')
```

Comment:
Might be better to have a test showing same groups with same options and same groups with different options.

Reply:
It makes sense! I updated this PR.